test/CMakeLists.txt

```diff
@@ -1,14 +1,8 @@
-cmake_minimum_required(VERSION 3.21)
-project(NESO VERSION 0.0.1)
-
-# GoogleTest requires at least C++11 DataParallel C++ requires at least C++17
-set(CMAKE_CXX_STANDARD 17)
-
-enable_testing()
-find_package(GTest QUIET)
+find_package(GTest REQUIRED)
 
 if(NOT GTest_FOUND)
   include(FetchContent)
```

Comment:
Is this block ever executed if `find_package(Gtest...)` is `REQUIRED`rather than `QUIET`?

Reply:
Ah, sorry, that change was only meant to be for debugging purposes. I'll revert it so GTest isn't required anymore.

Reply:
Fixed that now.